pkg/cloud/openstack/clients/networkservice.go

```diff
@@ -115,13 +116,22 @@ func (s *NetworkService) reconcileNetwork(networkName string, desired openstackc
 		return emptyNetwork, err
 	}
 
+	_, err = attributestags.ReplaceAll(s.client, "networks", network.ID, attributestags.ReplaceAllOpts{
+		Tags: []string{
+			"cluster-api-provider-openstack",
+			clusterName,
+		}}).Extract()
+	if err != nil {
+		return emptyNetwork, err
```

Comment:
I wonder if we should return the error or just ignore it. By returning it we're requiring OpenStack deployments to have this endpoint available (we'll be requiring newer versions of OpenStack).

FWIW, I prefer this but I wanted to make sure we're on the same page here.